pkg/cmd/pipelinerun/list.go

```diff
@@ -201,22 +208,33 @@ func reverse(prs *v1alpha1.PipelineRunList) {
 	prs.Items = prItems
 }
 
-func printFormatted(s *cli.Stream, prs *v1alpha1.PipelineRunList, c clockwork.Clock) error {
+func printFormatted(s *cli.Stream, prs *v1alpha1.PipelineRunList, c clockwork.Clock, allnamespaces bool) error {
 	if len(prs.Items) == 0 {
 		fmt.Fprintln(s.Err, emptyMsg)
 		return nil
 	}
 
 	w := tabwriter.NewWriter(s.Out, 0, 5, 3, ' ', tabwriter.TabIndent)
-	fmt.Fprintln(w, "NAME\tSTARTED\tDURATION\tSTATUS\t")
+	indexString := "NAME\tSTARTED\tDURATION\tSTATUS\t"
+	if allnamespaces {
+		indexString = indexString + "NAMESPACE\t"
+	}
+	fmt.Fprintln(w, indexString)
 	for _, pr := range prs.Items {
-
-		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t\n",
+		prNamespace := ""
+		formatString := "%s\t%s\t%s\t%s\t"
```

Comment:
I would look into your `formatString`. This is what is displaying locally if you run this:

```
NAME                      STARTED          DURATION     STATUS            NAMESPACE   
demo-pipeline-run-99s49   57 minutes ago   29 seconds   Failed   default     
demo-pipeline-run-1       1 hour ago       1 minute     Failed   default     

```

Reply:
I am not able to reproduce what you are seeing there.
```
NAME                           STARTED   DURATION   STATUS   NAMESPACE               
gcs-resource-2t859             ---       ---        ---      jenkins-operator-test   
gcs-resource-gst97             ---       ---        ---      tekton-pipelines        
task-to-list-files-run-v4crv   ---       ---        ---      tekton-pipelines 
```
It seems more of an issue with `indexString` as the distance between `STATUS` and `NAMESPACE` seems to be more than 1 tabspace.

Reply:
It happens for me as well 
```
NAME                                    STARTED      DURATION     STATUS                            NAMESPACE      
vote-api-buildah-18zk1t                 6 days ago   2 minutes    Failed                   test-local-2   
vote-api-buildah-pi79xy                 6 days ago   3 minutes    Succeeded                test-local ```

Reply:
Maybe we should think of using `template ` WDYT @danielhelfand

Reply:
Updated : using templates instead of string theory